project_app/Unit_Tests/Driver_test.py

```diff
@@ -37,8 +36,70 @@ def test_add_too_many_args(self):
 
 
 class TestEditAccount(TestCase):
-    def test_edit_acc(self):
+    def test_edit_Instructor(self):
         a = Driver()
+        a.addAccount("1", "Instructor@test.com", "pass1234", "test", "USA", "123-456-7890", 1)
+        self.assertEqual(a.editAccount("1", "2", "Instructor@test.com", "pass1234", "test", "USA", "123-456-7890", 1),
+                         ["", "", "", "", "", "", ""])
+
+    def test_edit_TA(self):
+        a = Driver()
+        a.addAccount("3", "TA@test.com", "pass1234", "test", "USA", "123-456-7890", 2)
+        self.assertEqual(a.editAccount("3", "4", "TA@test.com", "pass1234", "test", "USA", "123-456-7890", 2),
+                         ["", "", "", "", "", "", ""])
+
+    def test_edit_Supervisor(self):
+        a = Driver()
+        a.addAccount("5", "Supervisor@test.com", "pass1234", "test", "USA", "123-456-7890", 0)
+        self.assertEqual(a.editAccount("5", "6", "Supervisor@test.com", "pass1234", "test", "USA", "123-456-7890", 0),
+                         ["", "", "", "", "", "", ""])
+
+    def test_add_wrong_type(self):
+        a = Driver()
+        with self.assertRaises(TypeError, msg="invalid argument"):
+            a.editAccount(1, 1, 1, 1, 1, 1, 1, 1, 1)
+
+    def test_add_too_few_args(self):
+        a = Driver()
+        with self.assertRaises(TypeError, msg="too few arguments"):
+            a.editAccount("test")
+
+    def test_add_too_many_args(self):
+        a = Driver()
+        with self.assertRaises(TypeError, msg="too many arguments"):
+            a.editAccount("", "", "", "", "", "", "", "", "", 0)
+
+
+class TestDeleteAccount(TestCase):
+    def test_delete_Instructor(self):
+        a = Driver()
+        a.addAccount("24", "Instructor@test.com", "pass1234", "test", "USA", "123-456-7890", 1)
+        self.assertIsNone(a.deleteAccount(24))
```

Comment:
So this just checks that deleteAccount returned None (means nothing since it doesn't even have a return statement) to check if it was deleted from the database you can do:
with self.assertRaises(UserModel.DoesNotExist, msg=""):
    UserModel.objects.get(user_id=24)